inc/osvr/Client/LocateServer.h

```diff
@@ -42,6 +42,15 @@ namespace osvr {
         /** @brief INTERNAL ONLY - get the current server directory, if available. */
         inline boost::optional<std::string> getServerBinaryDirectoryPath() {
             auto server = osvr::common::getEnvironmentVariable("OSVR_SERVER_ROOT");
+            if (server) {
+                std::string pathSep;
+                if (server->size() > 0) {
+                    char lastChar = server->at(server->size() - 1);
+                    if (lastChar == '/' || lastChar == '\\') {
+                        return server->substr(0, server->size() - 1);
```

Comment:
You can use `server->pop_back();` in C++11 to remove the last char from the string.